ksqldb-engine/src/main/java/io/confluent/ksql/internal/PullQueryExecutorMetrics.java

```diff
@@ -82,10 +93,19 @@ public PullQueryExecutorMetrics(
     this.localRequestsSensor = configureLocalRequestsSensor();
     this.remoteRequestsSensor = configureRemoteRequestsSensor();
     this.latencySensor = configureRequestSensor();
+    this.latencySensorMap = configureRequestSensorMap();
     this.requestRateSensor = configureRateSensor();
     this.errorRateSensor = configureErrorRateSensor();
+    this.errorRateSensorMap = configureErrorSensorMap();
     this.requestSizeSensor = configureRequestSizeSensor();
     this.responseSizeSensor = configureResponseSizeSensor();
+    this.responseSizeSensorMap = configureResponseSizeSensorMap();
+    this.responseCode2XX = configureStatusCodeSensor("2XX");
```

Comment:
Just curious, why don't we further categorize on source/plan/forward in addition to error codes?

Reply:
We could.  These metrics just overlap significantly with some of the others like the error or request ones (which are broken down) and I didn't want to break every little thing down if it wasn't necessary.  For these high level status codes, I figure we just want to generally keep tabs and can dig down further with the other metrics if necessary.